### ISSUE TITLE
protein modification regions [sf#108]

### ISSUE THREAD

**Issue by srynobio** [The-Sequence-Ontology/SO-Ontologies#108]
**Reported by eilbeck on 2008-07-25 17:15 UTC**
I am adding this to the tracker as I think Robin brought up a good point about capturing the region where a modification occurs - which is slightly different from the modification itself.  This discussion came from a request for terms to describe the GPI anchor. The suggestion is to make cross products with psi-mod.
\--K

On 7/21/08 4:33 AM, "Robin Houston" &lt;rh11@sanger.ac.uk&gt; wrote:

Interesting. Is it planned to move some of the SO terms into the PSI-
MOD ontology, then? It would be useful to see an explanation of the
\(intended?\) formal relationship between the two ontologies, if there
is such a thing.

To my naive eyes, it's not altogether clear how this fits what we're
looking for. MOD:00818 \(glycosylphosphatidylinositolated residue\) is\_a
MOD:00000 \(protein modification\), whereas we would like a term to
describe a feature of the polypeptide \(the GPI anchor\). Is it planned
to introduce some cross-product terms into SO to make this possible
somehow?

Thanks,

Robin

On 17 Jul 2008, at 10:55, Sandra Orchard wrote:

&gt; Hi Christian
&gt;
&gt; Sorry about the delay in getting back to you, but the status at the
&gt; moment is
&gt;
&gt; GPI\_anchor is already present \(MOD:00818\)
&gt; I have spoken to John Garavelli about including the cleavage site in
&gt; the MOD ontology as it really is a PTM - I'll let you know his
&gt; decision.
&gt;
&gt; UniProtKB annotate the cleaved region as a propeptide, removed in
&gt; the mature form and I agree with Gabby that this is covered by
&gt; cleaved\_membrane\_spanning\_domain. You can use the label feature to
&gt; add more explanation if you wish.
&gt;
&gt; I hope this goes someway towards dealing with your query and I'll
&gt; get back to you as soon as I have heard from John.
&gt; Best wishes
&gt; Sandra
&gt;
&gt;
&gt; Gabrielle Reeves wrote:
&gt;&gt; Dear Christiane,
&gt;&gt;
&gt;&gt; Apologies for the multiple emails...  I am leaving the EBI and
&gt;&gt; therefore I am currently in the process of handing over the
&gt;&gt; responsibility of the protein feature terms to my colleague Sandra
&gt;&gt; Orchard, who is CCd on this email.
&gt;&gt;
&gt;&gt; Sandra will be very pleased to discuss these terms with you.
&gt;&gt;
&gt;&gt; Regards
&gt;&gt;
&gt;&gt; Gabby
&gt;&gt;
&gt;&gt; Christiane Hertz-Fowler wrote:
&gt;&gt;&gt; Hi,
&gt;&gt;&gt; sounds good. as far as the position in the ontology is concerned,
&gt;&gt;&gt; is this where signal peptides sit as well?
&gt;&gt;&gt; Christiane
&gt;&gt;&gt; On 1 Jul 2008, at 12:11, Gabrielle Reeves wrote:
&gt;&gt;&gt;
&gt;&gt;&gt;&gt; Dear Christiane,
&gt;&gt;&gt;&gt;
&gt;&gt;&gt;&gt; I can sort these terms out but help for definitions and
&gt;&gt;&gt;&gt; classification would be greatly appreciated.
&gt;&gt;&gt;&gt;
&gt;&gt;&gt;&gt; Presumably we need to add these terms:
&gt;&gt;&gt;&gt;
&gt;&gt;&gt;&gt; GPI\_anchor
&gt;&gt;&gt;&gt; GPI\_anchor\_cleavage\_site
&gt;&gt;&gt;&gt; cleaved\_membrane\_spanning\_domain \(we already have
&gt;&gt;&gt;&gt; cleaved\_peptide\_region - maybe this works?\)
&gt;&gt;&gt;&gt;
&gt;&gt;&gt;&gt; I am slightly unclear of the exact biology.
&gt;&gt;&gt;&gt;
&gt;&gt;&gt;&gt; Can they all sit as a child of "immature\_peptide\_region -&gt;
&gt;&gt;&gt;&gt; cleaved\_peptide\_region"
&gt;&gt;&gt;&gt;
&gt;&gt;&gt;&gt;
&gt;&gt;&gt;&gt; Gabby
&gt;&gt;&gt;&gt;
&gt;&gt;&gt;&gt;
&gt;&gt;&gt;&gt;
&gt;&gt;&gt;&gt; Karen Eilbeck wrote:
&gt;&gt;&gt;&gt;&gt; Hi Christiane,
&gt;&gt;&gt;&gt;&gt; This term does not exist currently, but we can make it for.
&gt;&gt;&gt;&gt;&gt; Would you call it GPI\_anchor\_cleavage\_site?
&gt;&gt;&gt;&gt;&gt; I am hoping that Gabby is still around to help with this...
&gt;&gt;&gt;&gt;&gt; Would you need a term also for the part of the protein that is
&gt;&gt;&gt;&gt;&gt; cleaved and left in the membrane? What would the name be?
&gt;&gt;&gt;&gt;&gt; Cleaved\_membrane\_spanning\_doman?
&gt;&gt;&gt;&gt;&gt; Could you both think about the position in the ontology and
&gt;&gt;&gt;&gt;&gt; definitions for these terms? I can help too, but Gabby is in a
&gt;&gt;&gt;&gt;&gt; better position for this.
&gt;&gt;&gt;&gt;&gt; Thanks,
&gt;&gt;&gt;&gt;&gt; Karen
&gt;&gt;&gt;&gt;&gt; On 6/30/08 11:33 AM, "Christiane Hertz-Fowler"
&gt;&gt;&gt;&gt;&gt; &lt;chf@sanger.ac.uk&gt; wrote:
&gt;&gt;&gt;&gt;&gt;    Hi,
&gt;&gt;&gt;&gt;&gt;    I am trying to find the appropriate SO term to describe the
&gt;&gt;&gt;&gt;&gt; cleavage
&gt;&gt;&gt;&gt;&gt;    site for GPI anchor addition ... doe such a term currently
&gt;&gt;&gt;&gt;&gt; exist in
&gt;&gt;&gt;&gt;&gt;    the ontology?
&gt;&gt;&gt;&gt;&gt;    Regards,
&gt;&gt;&gt;&gt;&gt;    Christiane
&gt;&gt;&gt;&gt;&gt;    --
&gt;&gt;&gt;&gt;&gt;     The Wellcome Trust Sanger Institute is operated by Genome
&gt;&gt;&gt;&gt;&gt; Research
&gt;&gt;&gt;&gt;&gt;     Limited, a charity registered in England with number 1021457
&gt;&gt;&gt;&gt;&gt; and a
&gt;&gt;&gt;&gt;&gt;     company registered in England with number 2742969, whose
&gt;&gt;&gt;&gt;&gt; registered
&gt;&gt;&gt;&gt;&gt;     office is 215 Euston Road, London, NW1 2BE.
&gt;&gt;&gt;&gt;&gt;

**Comment by srynobio** [comment]
**Commented by eilbeck on 2008-08-07 21:25 UTC**
Logged In: YES 
user\_id=742851
Originator: YES

Hi Robin,
I have added 21 modified amino acid terms as children of amino acid to SO.
These terms can be used to annotate which feature \(residue\) is modified although the exact type of modification would still need to be specified further by another resource devoted to modifications such as psi-mod. 
I have added  cross references for these terms to psi-mod, and included their names and synonyms as SO synonyms. I also created a new synonym category amino acid modification to distinguish these synonyms.
There are no plans to make cross products at this time.

Thanks for pointing this out.
\--Karen
